examples/distributed-primitives/idgenerator/FlakeIdGenerator.cpp

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2008-2018, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include <hazelcast/client/HazelcastClient.h>
+#include <hazelcast/client/FlakeIdGenerator.h>
+
+int main() {
+    hazelcast::client::ClientConfig config;
+    hazelcast::client::HazelcastClient hz(config);
+
+    hazelcast::client::FlakeIdGenerator generator = hz.getFlakeIdGenerator("flakeIdGenerator");
+    for (int i = 0; i < 10000; ++i) {
+        hazelcast::util::sleep(1);
```

Comment:
Is this in milliseconds ?

Reply:
that is seconds. sleepmillis for milliseconds.